images/centos/7/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM docker.io/library/centos:7
+
+ENV NAME=centos-toolbox VERSION=7
+LABEL com.github.debarshiray.toolbox="true" \
```

Comment:
Maybe update this based on https://github.com/containers/toolbox/commit/de5e5df9b793edc746cde7125cd437b69f15ccd7 while waiting for further review